pkg/apis/batch/types.go

```diff
@@ -39,6 +39,15 @@ type Job struct {
 	Status JobStatus `json:"status,omitempty"`
 }
 
+func (j *Job) IsJobFinished() bool {
```

Comment:
Additionally make it accept Job as a parameter, rather than working on the Job object itself.